app/routes/surveys/components/Submissions/SubmissionsIndividual.tsx

```diff
@@ -1,15 +1,11 @@
 import cx from 'classnames';
+import { useOutletContext } from 'react-router-dom';
 import StaticSubmission from '../StaticSubmission';
 import styles from '../surveys.css';
-import type { SelectedSurvey } from 'app/reducers/surveys';
-import type { SurveySubmission } from 'app/store/models/SurveySubmission';
+import type { ContextType } from '../../index';
```

Comment:
```suggestion
import type { ContextType } from 'app/routes/surveys';
```
Easier to read absolute imports, especially with multiple `../`s